server/adaptors/integrations/integrations_manager.ts

```diff
@@ -157,6 +157,8 @@ export class IntegrationsManager {
     templateName: string,
     name: string,
     indexPattern: string,
+    dataSourceMDSId: string | undefined,
+    dataSourceMDSLabel: string | undefined,
```

Comment:
Too small to worry about now, but I notice there's a lot of `var: string | undefined` instead of `var?: string`. I'll probably end up rewriting these next time I'm messing with this code since the inconsistent syntax bugs me.

Reply:
I will change it in the cases I have written